app/src/main/res/xml/appfilter.xml

```diff
@@ -2507,6 +2526,10 @@
     <!-- Radio -->
     <item component="ComponentInfo{com.tct.fmradio.bb/com.tct.fmradio.utils.BaseActivity}" drawable="radio" />
 
+    <!-- raiffeisenbank -->
+    <!-- eKonto -->
```

Comment:
There seems to be an issue here, there should only be one comment per group.

Reply:
Raiffeisenbank is the bank name and eKonto is the app name. because there are several Raiffeisenbanks in various countries in the world, [each having its own app](https://play.google.com/store/search?q=raiffeisen&c=apps) with similar logo. I thought we could next them as they could potentially use the same icon.

Reply:
I moved this entry under `eKonto` only and kept the bank name for the icon.